src/components/button/utils.ts

```diff
@@ -0,0 +1,16 @@
+export function buttonIntentToColor(intent: string) {
+  switch (intent) {
+    case 'primary':
+      return 'white';
+    case 'secondary':
+      return 'primary';
+    case 'ghost':
+      return 'primary';
+    case 'ghost':
+      return 'primary';
+    case 'alert':
+      return 'labels.red';
+    default:
+      return 'primary';
+  }
+}
```

Comment:
neat: would be more beautiful if you use objects here as well as you did in other places